src/pages/Faucet/FaucetModal.tsx

```diff
@@ -1,9 +1,9 @@
 import Modal from "../../components/Modal";
-import { shortenHash } from "../../utils/formatting";
+import { shortenAddress } from "../../utils/formatting";
 type Props = {
   isOpen: boolean;
   onClose: () => void;
-  txHash?: string | undefined;
+  txHash?: string;
```

Comment:
why is the txHash undefined? dont we want the modal to show if there is a tx hash?

Reply:
Good question that's what I thought too. By looks of it, the faucet modal is always open. It's just not being displayed until a claim is done. Due to this, it is receiving an undefined txHash most of the time.

Reply:
I just thought I'd change that line because it's not necessary to have an undefined type with a ? - it's not related to this ticket's functionality

Reply:
I have changed some logic to fix this